Source/Orts.Simulation/Simulation/AIs/AI.cs

```diff
@@ -833,6 +834,8 @@ private AITrain CreateAITrain(Service_Definition sd, Traffic_Traffic_Definition
                 StartList.InsertTrain(train);
                 Simulator.StartReference.Add(train.Number);
             }
+//            if (train.Cars[0] is MSTSLocomotive && (train.Cars[0] as MSTSLocomotive).EOTEnabled != MSTSLocomotive.EOTenabled.no)
+//                    train.EOT = new EOT((train.Cars[0] as MSTSLocomotive).EOTEnabled, true, train);
```

Comment:
Is this commented code needed or can be removed?

Reply:
It may be removed.